bigframes/_config/bigquery_options.py

```diff
@@ -235,3 +239,8 @@ def kms_key_name(self, value: str):
             raise ValueError(SESSION_STARTED_MESSAGE.format(attribute="kms_key_name"))
 
         self._kms_key_name = value
+
+    @property
+    def _strictly_ordered(self) -> bool:
```

Comment:
Presumably we'll have a public version of this at some future date?

Reply:
Yes, this is just a private placeholder until the feature is complete enough to expose through a public API. We will also need to settle on the naming.